sdk/maps/azure-maps-route/README.md

```diff
@@ -0,0 +1,242 @@
+```
+
+### Get Route Matrix
+
+If the Matrix Route request was accepted successfully, the Location header in the response contains the URL to download the results of the request.
+
+Retrieves the result of a previous route matrix request.
+The method returns a poller for retrieving the result.
+
+```python
+from azure.maps.route import MapsRouteClient
+
+route_matrix_result = client.begin_get_route_matrix_result(matrix_id="11111111-2222-3333-4444-555555555555");
+```
+
+### Get Route Directions Batch
+
+Retrieves the result of a previous route direction batch request.
+The method returns a poller for retrieving the result.
+
+```python
+from azure.maps.route import MapsRouteClient
+
+route_directions_batch_poller_result = client.begin_get_route_directions_batch_result(batch_id="11111111-2222-3333-4444-555555555555");
+```
+
+## Troubleshooting
+
+### General
+
+Maps Route clients raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/README.md).
+
+This list can be used for reference to catch thrown exceptions. To get the specific error code of the exception, use the `error_code` attribute, i.e, `exception.error_code`.
+
+### Logging
+
+This library uses the standard [logging](https://docs.python.org/3/library/logging.html) library for logging.
+Basic information about HTTP sessions (URLs, headers, etc.) is logged at INFO level.
+
+Detailed DEBUG level logging, including request/response bodies and unredacted headers, can be enabled on a client with the `logging_enable` argument:
+
+```python
+import sys
+import logging
+from azure.maps.route import MapsRouteClient
+
+# Create a logger for the 'azure.maps.route' SDK
+logger = logging.getLogger('azure.maps.route')
+logger.setLevel(logging.DEBUG)
+
+# Configure a console output
+handler = logging.StreamHandler(stream=sys.stdout)
+logger.addHandler(handler)
+
+```
+
+Similarly, `logging_enable` can enable detailed logging for a single operation,
+even when it isn't enabled for the client:
+
+```python
+service_client.get_service_stats(logging_enable=True)
+```
+
+### Additional
+
+Still running into issues? If you encounter any bugs or have suggestions, please file an issue in the [Issues](<https://github.com/Azure/azure-sdk-for-python/issues>) section of the project.
+
+## Next steps
+
+### More sample code
+
+Get started with our [Maps Route samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-route/samples) ([Async Version samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-route/samples/async_samples)).
+
+Several Azure Maps Route Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Maps Route
+
+```bash
+set AZURE_SUBSCRIPTION_KEY="<RealSubscriptionKey>"
+
+pip install azure-maps-route --pre
```

Comment:
We have `--pre` in Route PR but not in Search PR. Do you think we can sync the description in these two PRs?

Reply:
Aligned with in Search SDK.